modules/client/src/internal/client/methods.ts

```diff
@@ -1110,12 +1109,12 @@ export class ClientMethods extends ClientCore implements IClientMethods {
     return version;
   }
 
-  public isDaoUpdateProposal(
+  public isDaoUpdate(
     actions: DaoAction[],
   ): boolean {
     const initializeFromInterface = DAO__factory.createInterface()
       .getFunction("initializeFrom").format("minimal");
-    return findAction(actions, initializeFromInterface) !== -1;
+    return findActionIndex(actions, initializeFromInterface) !== -1;
```

Comment:
I think I've found something here: There will be cases in which we update the DAO **without reinitializing** and thus without calling `initializeFrom`. This method of checking for DAO updates is therefore not sufficient. Shouldn't we rather look for `upgradeTo` or `upgradeToAndCall` in the actions?
@mathewmeconry @Rekard0

Reply:
we can address it here: https://aragonassociation.atlassian.net/browse/OS-777